src/script/view_model/ActionsViewModel.ts

```diff
@@ -64,6 +67,14 @@ export class ActionsViewModel {
     return this.conversationRepository.archiveConversation(conversationEntity);
   };
 
+  readonly unArchiveConversation = (conversationEntity: Conversation): void => {
```

Comment:
```suggestion
  readonly unarchiveConversation = (conversationEntity: Conversation): void => {
```

I believe the convention is to consider `unarchive` as a single word :)